Kitodo/src/main/java/de/sub/goobi/helper/BeanHelper.java

```diff
@@ -186,17 +187,17 @@ public static void copyProperties(Process processTemplate, Process processCopy)
      * @param processCopy
      *            new object
      */
-    public static void copyScanTemplates(Process processTemplate, Process processCopy) {
+    public static void copyScanTemplates(Template processTemplate, Process processCopy) {
         List<Property> templateProperties = new ArrayList<>();
-        for (Property templateProperty : processTemplate.getTemplates()) {
+        /*for (Property templateProperty : processTemplate.getTemplates()) {
```

Comment:
Why commenting you this out?

Reply:
I'm waiting for decision if it can be fully removed.

Reply:
It is necessary to make it now, as I want to work further on workflow and this PR contains breaking changes. Maybe I can just remove it as more likely that will be the final decision?

Reply:
Removed.